doc/man/scons.xml

```diff
@@ -6414,18 +6414,25 @@ env.Command('marker', 'input_file', action=[MyBuildAction, Touch('$TARGET')])
 &scons;
 performs variable substitution on the string that makes up
 the action part of the builder.
-Variables to be interpolated are indicated in the
-string with a leading
-<literal>$</literal>, to distinguish them from plain text
+Variables or other text to be substituted are indicated in the
```

Comment:
When would other text work? $OTHER_TEXT when not a variable is just blank..

Reply:
the planned followon will improve on the way this is structured... things like a python expression are not variables, thus the "other"

Reply:
but python expressions are only valid inside ${ }, and not $xyz

Reply:
Yes, that's what I meant by a restructure in a future revision.  I plan to propose it as a substitutable is `${text}` but under these circumstances you can leave out the `{}`, which I think makes it more consistent,  I can pull out the "or other text" from here for now if that makes you happier, don't know when I'll get to finishing up the next round.